DevFullstackCo/Crypto-Voting-Project
# Add About page design and content updates

### Summary:
This pull request adds the About page for PulseCoinTribe, with a focus on:
- Dynamic and engaging text describing the platform's purpose and features.
- A professional and futuristic design consistent with the app's overall aesthetic.
- A new image representing the platform's community spirit and market insights.

### Changes:
1. Added an `about.html.erb` template in `app/views/static_pages/`.
2. Styled the About page with `about.css` located in `app/assets/stylesheets/`.
3. Added a new image (`crypto-community.png`) to `app/assets/images/`.

### How to Test:
- Navigate to the About page (`http://localhost:3000/about`) to verify layout and functionality.
- Check responsiveness and consistency with the overall app design.

### Notes:
This page establishes the foundation for presenting the platform's mission and engaging potential users.
